project_4/styles.css

```diff
@@ -0,0 +1,667 @@
+    font-size: 18px;
+}
+
+.speaker-paragraph {
+    font-family: var(--font-paragraph);
+}
+
+/* hover effects for speakers */
+.speaker-info img:hover {
+    border: 8px dashed var(--color-blue);
+}
+
+
+/* =============
+SCHEDULE
+================ */
+
+.schedule-info,
+.schedule-time {
+    font-family: var(--font-paragraph);
+}
+
+.schedule-info {
+    border: 4px dashed var(--color-blue);
+    list-style-type: none;
+    margin: auto;
+    padding: 10px 20px;
+}
+
+.schedule-time {
+    background-color: var(--color-green);
+    color: white;
+    display: inline-block;
+    margin: 30px 0 10px;
+    padding: 7px 30px;
+    text-transform: uppercase;
+}
+
+.schedule-title {
+    font-weight: bold;
+    text-transform: uppercase;
+}
+
+/* special talk colors */
+.talk-header {
+    border: 4px dashed var(--color-brown);
+}
+
+.talk-time {
+    background-color: var(--color-brown);
+}
+
+/*  */
+
+.left-friday,
+.right-saturday {
+    margin: auto;
+    width: 80%;
+}
+
+h4 {
+    border-left: 5px var(--color-blue) solid;
+    font-family: var(--font-header);
+    font-size: 20px;
+    padding: 10px;
+    text-transform: uppercase;
+}
+
+/* ================
+REGISTER SECTION
+================== */
+
+.register {
+    background-color: var(--color-green);
+    margin-top: 100px;
+    padding: 100px 30px;
+    text-align: center;
+}
+
+.register a {
+    border: 1px dashed var(--color-offwhite);
+    color: white;
+    font-family: var(--font-header);
+    font-size: 30px;
+    padding: 10px;
+    text-decoration: none;
+    text-transform: uppercase;
+    width: 200px;
+}
+
+.register-text {
+    color: white;
+    font-family: var(--font-header);
+    font-size: 18px;
+    margin-bottom: 40px;
+}
+
+/* hover effects for register button */
+.register a:hover {
+    background-color: var(--color-brown);
+}
+
+/* ===============
+FOOTER
+================= */
+
+.footer {
+    background-color: var(--color-brown);
+    height: 200px;
+    margin-top: 150px;
+    padding: 60px 40px 10px;
+}
+
+/* social media icons */
+.social-media-icons a {
+    display: inline-block;
+    fill: var(--color-blue);
+    padding: 15px;
+    transition: transform .2s;
+    width: 40px;
+}
+
+.social-media-icons a:hover {
+    fill: var(--color-green);
+    transform: scale(1.5);
+}
+
+.follow-us {
+    color: white;
+    max-width: 100%;
+}
+
+.copyright {
+    color: var(--color-offwhite);
+    font-size: 9px;
+}
+
+.follow-us,
+.copyright {
+    font-family: var(--font-paragraph);
+}
+
+
+/* ==============
+MEDIA QUERIES
+================= */
+
+@media (max-width: 420px) {
+    .register a {
+        font-size: 26px;
+    }
+}
+
+@media (min-width: 600px) {
+    .speakers {
+        width: 50%;
+    }
+
+    nav ul a {
+        font-size: 17px;
+        padding: 30px 20px;
+    }
+}
+
+
+@media (min-width: 800px) {
+
+    /* header 2 */
+    .section-header {
+        font-size: 70px;
+    }
+
+    /* navigation */
+    nav ul a {
+        font-size: 17px;
```

Comment:
Since this is the same as your previous media query, you don't need to reiterate it -- the previous media query is effect starting at 600px and until infinity.
```suggestion
```